woocommerce-gateway-stripe.php

```diff
@@ -225,6 +225,52 @@ public function install() {
 
 					add_woocommerce_inbox_variant();
 					$this->update_plugin_version();
+
+					// TODO: Remove this when we're reasonably sure most merchants have had their
+					// settings updated like this. ~80% of merchants is a good threshold.
```

Comment:
how do we know what amount of users have migrated to `5.4.0` or newer?

Reply:
We have some numbers on the spread of Stripe plugin versions in use that we can fall back on here 🙂